dsc/docs-conceptual/dsc-3.0/reference/schemas/resource/manifest/root.md

```diff
@@ -249,22 +291,23 @@ resource. This property must always be an object that defines one of the followi
 - `embedded` - When you specify the `embedded` property, DSC uses the defined value as the JSON
   schema.
 
-For more information, see [DSC Resource manifest schema property reference][12].
+For more information, see [DSC Resource manifest schema property reference][13].
 
 ```yaml
 Type:     object
 Required: true
 ```
 
 [01]: ../../definitions/resourceType.md
-[02]: ../../../cli/config/export.md
-[03]: ../../../cli/resource/export.md
-[04]: ../../../cli/resource/get.md
-[05]: ../../../cli/resource/get.md#-a---all
-[06]: export.md
-[07]: get.md
-[08]: set.md
-[09]: test.md
-[10]: validate.md
-[11]: provider.md
-[12]: schema/property.md
+[02]: ../../definitions/resourceKind.md
+[03]: ../../../cli/config/export.md
+[04]: ../../../cli/resource/export.md
+[05]: ../../../cli/resource/get.md
+[06]: ../../../cli/resource/get.md#-a---all
+[07]: export.md
+[08]: get.md
+[09]: set.md
+[10]: test.md
+[11]: validate.md
+[12]: provider.md
```

Comment:
You link to `provider.md` but this PR deletes that file.

Reply:
```suggestion
[12]: adapter.md
```